test/unit/lint/test-licenses.js

```diff
@@ -0,0 +1,96 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import path from 'path';
+import expect from 'expect';
+import checker from 'license-checker';
+import { autoFailingAsyncTest } from '../../utils/async';
+const root = path.join(__dirname, '..', '..', '..');
+
+const WAIVED_DEPENDENCIES = [
+  // tweetnacl seems to be OK, in Public Domain
+  // https://github.com/dchest/tweetnacl-js/blob/master/COPYING.txt
+  'tweetnacl@0.14.3',
+];
+
+const VALID_LICENSES = [
+  'ISC',
+
+  'MIT',
+  'MIT*',
+  'MIT/X11',
+
+  'CC-BY-3.0',
+
+  'AFLv2.1',
+
+  'LGPL',
+  'GPLv2',
+  'GPLv3',
+
+  'MPLv2.0',
+
+  'Apache',
+  'Apache*',
+  'Apache2',
+  'Apache License, Version 2.0',
+  'Apache-2.0',
+
+  'BSD',
+  'BSD-2-Clause',
+  'BSD-3-Clause',
+  'BSD-like',
+  'BSD*',
+
+  'EPL',
+
+  'Artistic-2.0',
+
+  // Questionable licenses
+  'WTFPL',
+  'Unlicense',
+  'Public Domain',
+  'Public domain',
+];
+
+function isValidLicense(license) {
```

Comment:
Not sure if something like http://npm.im/spdx would make this easier than maintaining and checking your own list if licenses.